pontoon/base/static/js/fluent_interface.js

```diff
@@ -258,7 +258,13 @@ var Pontoon = (function (my) {
 
         elements.forEach(function (item) {
           if (item.type === 'TextElement') {
-            translatedValue += item.value;
+            // TODO: We shouldn't rely on markPlaceables in determining when to not render HTML.
+            if (markPlaceables) {
+              translatedValue += Pontoon.doNotRender(item.value);
```

Comment:
@mathjazz is there a case when `value` shouldn't be escaped? If I understand correctly strings coming from db should be always escaped via `doNotRender()`

Reply:
Yes, we shouldn't escape in editor (textarea).